dataeng/jobs/analytics/AnswerDistribution.groovy

```diff
@@ -33,11 +33,12 @@ class AnswerDistribution {
                         string('OPSGENIE_HEARTBEAT_CONFIG_KEY', 'opsgenie_heartbeat_config_key')
                     }
                 }
+                publishers common_publishers(allVars)
                 steps {
                     shell(dslFactory.readFileFromWorkspace('dataeng/resources/opsgenie-enable-heartbeat.sh'))
                     shell(dslFactory.readFileFromWorkspace('dataeng/resources/answer-distribution.sh'))
-                    shell(dslFactory.readFileFromWorkspace('dataeng/resources/opsgenie-disable-heartbeat.sh'))
                 }
+                publishers opsgenie_heartbeat_publisher
```

Comment:
another Nit: put the publishers together.  They are grouped in the UI after the build steps, so I think it's reasonable to mirror that here.